README.md

```diff
@@ -153,15 +153,17 @@ iscsc.fr
 ├── frontend
 │   ├── public              *automatically generated files and images that are publically available for the user*
 │   └── src                 *source code of the website*
-│       ├── components/     *source code of main components of the website*
-│       ├── context/        *defines the context function to keep track data with useReducer*
-│       ├── hooks/          *defines the hooks that trigger the context functions*
-│       ├── pages/          *source code of the pages of the website*
-│       ├── App.js          *defines the routes of the application*
-│       ├── index.js        *main js application of the website*
-│       └── index.css       *css styling file of the website*
+│       ├── components/         *source code of main components of the website*
+│       ├── context/            *defines the context function to keep track data with useReducer*
+│       ├── hooks/              *defines the hooks that trigger the context functions*
+│       ├── pages/              *source code of the pages of the website*
+│       ├── App.js              *defines the routes of the application*
+│       ├── index.js            *main js application of the website*
+│       └── index.css           *css styling file of the website*
 │
-└── README.md               *this file*
+├── bump.sh             *script used to bump version of frontend, backend and whole website*
```

Comment:
I may but it would cause some problem because I must run `npm version..` while being in the right folder so I should add a `$PWD` check if I do this :wink:

Reply:
Mmm okay
But we have 3 incoming scripts from #34 #29 and this one 🤔 
So we may want to move it later
I'll resolve the thread but maybe open an issue later

Reply:
> But we have 3 incoming scripts from https://github.com/iScsc/iscsc.fr/pull/34 https://github.com/iScsc/iscsc.fr/pull/29 and this one thinking
>
#34 introduces a new script? :thinking: 

otherwise, i agree with @atxr

Reply:
seems fine to me!